jenkins/verify-chef.sh

```diff
@@ -10,24 +10,33 @@ export PATH
 EMBEDDED_BIN_DIR=/opt/$PROJECT_NAME/embedded/bin
 export EMBEDDED_BIN_DIR
 
+# If we are on Mac 10.11 our symlinks are located under /usr/local/bin
+# otherwise they are under /usr/bin
+if [[ $(uname -v) =~ ^Darwin\ Kernel\ Version\ 15 ]]; then
```

Comment:
We can simply this:

```bash
if test -f "/usr/bin/sw_vers"; then
```

Totally lifted from [Omnitruck's install.sh](https://github.com/chef/opscode-omnitruck/blob/84b9004fc8de26f5daff869400fffc26383d95d0/views/install.sh.erb#L139)